table/tables/tables.go

```diff
@@ -71,6 +72,11 @@ func TableFromMeta(alloc autoid.Allocator, tblInfo *model.TableInfo) (table.Tabl
 		}
 
 		col := table.ToColumn(colInfo)
+		if len(colInfo.GeneratedExprString) != 0 {
+			// here we can ignore error because GeneratedExprString has been already
+			// checked when this table or column was added.
+			col.GeneratedExpr, _ = parser.ParseExpression(colInfo.GeneratedExprString)
```

Comment:
We should check the error, it not worth to spend time thinking if it is safe to not check the error.